### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by curlyz** [nshoemaker/LowPowerRangers#1]
This project is so great, this  is AoA guys!! no one has done that publicly before

**Comment by curlyz** [comment]
@shashwatsrivastava94 @nshoemaker @critcher

**Comment by nshoemaker** [comment]
No. This was a project for a class that ended in May 2016. We are no longer actively working on it.
